src/frontend/src/pages/DRCPage/DRCPage.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+import { useState } from 'react';
+import { Box, Grid, ListItemIcon, Menu, MenuItem, Typography, useTheme } from '@mui/material';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import { NERButton } from '../../components/NERButton';
+import NoteAddIcon from '@mui/icons-material/NoteAdd';
+import PageLayout from '../../components/PageLayout';
+import { DesignReview, DesignReviewStatus } from 'shared';
+import MonthSelector from './MonthSelector';
+import DayCard from './DayCard';
+import FillerCard from './FillerCard';
+import { batman } from '../../../../backend/tests/test-data/users.test-data';
+
+const DRCPage = () => {
+  const theme = useTheme();
+
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [displayMonth, setDisplayMonth] = useState<Date>(new Date());
+
+  const EventDict = new Map<Number, DesignReview[]>();
+  // Test data:
+  EventDict.set(new Date().getDate(), [
+    {
+      designReviewId: 'Meeting',
+      dateScheduled: new Date(),
+      meetingTimes: [16],
+      dateCreated: new Date(),
+      userCreated: batman,
+      status: DesignReviewStatus.UNCONFIRMED,
+      teamType: { teamTypeId: 'Mechanical', name: 'Mechanical' },
+      requiredMembers: [],
+      optionalMembers: [],
+      confirmedMembers: [],
+      deniedMembers: [],
+      isOnline: false,
+      isInPerson: false,
+      attendees: [],
+      wbsName: 'bruh',
+      wbsNum: { carNumber: 1, workPackageNumber: 1, projectNumber: 1 }
+    }
+  ]);
+  console.log(EventDict);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleDropdownClose = () => {
+    setAnchorEl(null);
+  };
+
+  const daysInMonth = (month: Date) => {
+    return new Date(month.getFullYear(), month.getMonth() + 1, 0).getDate();
+  };
+
+  const paddingDays = (month: Date) => {
+    return new Date(month.getFullYear(), month.getMonth(), 0).getDay();
+  };
+
+  const paddingArrayStart = [...Array<number>(paddingDays(displayMonth)).keys()].map(
+    (day) => daysInMonth(new Date(displayMonth.getDate(), displayMonth.getMonth() - 1, displayMonth.getFullYear())) - day
+  );
+  const paddingArrayEnd = [...Array<number>(7 - ((daysInMonth(displayMonth) + paddingDays(displayMonth)) % 7)).keys()].map(
+    (day) => day + 1
+  );
+  const daysThisMonth = paddingArrayStart
+    .concat([...Array(daysInMonth(displayMonth)).keys()])
+    .map((day) => day + 1)
+    .concat(paddingArrayEnd.length < 7 ? paddingArrayEnd : []);
+
+  const unconfirmedDRSDropdown = (
```

Comment:
you should be able to use the `ActionsMenu` component here to simplify a lot of this code